### PR TITLE
Fix interest done not triggering.

### PR DESCRIPTION
**Pull request by Gamenot**
A mistake that should have been obvious.

### CODE REVIEW

**Review by qianyi-sun — APPROVED** [review]
have merged this branch on my branch and tested with several scenarios, the episodes now end with the vehicle of interest done, thanks Tucker!

File: scenarios/sumo/loop/scenario.py [hunk 1]
```diff
@@ -62,6 +68,7 @@
                 follow_offset=(-7, 10),
             ),
         ],
+        scenario_metadata=t.ScenarioMetadata(r"(leader)|(target)", Colors.Blue),
```
Comment (Adaickalavan) [inline scenarios/sumo/loop/scenario.py:71]: target or other_interest?

File: smarts/core/smarts.py [hunk 2]
```diff
@@ -1423,7 +1423,7 @@ def _process_collisions(self):
                 actor_id = self._vehicle_index.owner_id_from_vehicle_id(collidee.id)
                 # TODO: Should we specify the collidee as the vehicle ID instead of
                 #       the agent/social ID?
-                collision = Collision(collidee_id=actor_id)
+                collision = Collision(collidee_id=actor_id or collidee.id)
```
Comment (Adaickalavan) [inline smarts/core/smarts.py:1426]: Perhaps choose one instead of using `or`.

**Review by Adaickalavan — APPROVED** [review]

File: smarts/core/smarts.py (hunk 2 above)
Comment (Gamenot) [reply smarts/core/smarts.py:1426]: @Adaickalavan do you prefer knowing the agent id (or other type of owner) or the vehicle id?

Comment (Gamenot) [reply smarts/core/smarts.py]: I have made it always the vehicle name.

**Pull request merged**